cxxheaderparser/types.py

```diff
@@ -298,6 +301,20 @@ def format_decl(self, name: str):
         return f"{c}{v}{self.typename.format()} {name}"
 
 
+@dataclass
+class Operator:
+    """An internal structure for parsing operator."""
+
+    pqname: PQName
+    """Possibly qualified name for operator."""
+    operator_name: str
+    """Conversion operator have always `conversion` str in this attribute."""
+    ctype: Type
+    """Return type for this operator."""
+    cmods: "ParsedTypeModifiers"
+    """Return type modifiers for this operator."""
+
```

Comment:
This is only for internal usage to store data between `_parse_type` and `_parse_operator_conversion` in `_parse_declarations` method.